packages/camera/index.ios.ts

```diff
@@ -196,9 +196,9 @@ export let isAvailable = function () {
 
 export let requestPermissions = function () {
 	return new Promise(function (resolve, reject) {
-		requestPhotosPermissions().then(() => {
-			requestCameraPermissions().then(resolve, reject);
-		}, reject);
+		// Even if we don't have photo access we may want to get camera access.
```

Comment:
I don't agree with ignoring the failure to get photo permissions, if the user does not want photo permissions ( because they are not saving to the gallery)  they do not have to call this method, ignoring the failure removes the ability to inform the user, possibly directing them to enable permissions.

Reply:
I understand. But:
* I don't like to have a result that relies on the internal implementation (requesting first photos or first permissions)
* I don't like to have a failure without knowing which permission was faulty

So, do we want to have something like [@nativescript-community/perms](https://github.com/nativescript-community/perms/blob/master/src/perms/index.d.ts) that would return a `Promise<MultiResult>`.

Or something simpler returning just a boolean (granted or not)

```
requestCameraPermissions(): Promise<0 | 1>
requestPhotosPermissions(): Promise<0 | 2>
requestPermissions(): Promise<number>
enum TypeFlags {
    Camera = 1,
    Photo = 2,
}
```